classes/Visualizer/Source/Query.php

```diff
@@ -80,7 +80,7 @@ public function fetch( $as_html = false, $results_as_numeric_array = false, $raw
 		}
 
 		// only select queries allowed.
-		if ( preg_match( '/^\s*(insert|delete|update|replace|create|alter|drop|truncate)\s/i', $this->_query ) ) {
+		if ( preg_match( '/\s*(insert|delete|update|replace|create|alter|drop|truncate)\s/i', $this->_query ) ) {
```

Comment:
This might create a problem for users whose fields contain one of the strings in the group(like fields with name `last_update`). Example: https://regex101.com/r/lhDZe4/3

I suggest removing the `*` to the regex so that we can only find isolated keywords https://regex101.com/r/jwBxH5/1